lib/std/monotimes.nim

```diff
@@ -34,6 +34,7 @@ that the actual supported time resolution differs for different systems.
 See also
 ========
 * `times module <times.html>`_
+* `cputicks module <times.html>`_ which provides strictly monotonic cpu counter
```

Comment:
It's probably worth it to also add it to `docs/lib.rst`, under the "Time handling" section.

Reply:
done